src/components/ZclCreateModifyEndpoint.vue

```diff
@@ -254,7 +255,11 @@ export default {
     } else {
       this.shownEndpoint.endpointIdentifier = this.getSmallestUnusedEndpointId()
     }
-
+    const enableZigbeeFeatures =
```

Comment:
Same comment as earlier. Please stop polluting UI code with "zigbee" and "matter".

Reply:
so instead of doing a high level conditional that we can derive data from, you suggest doing a conditional for every single component?